src/core/trace/internalTracer.cpp

```diff
@@ -49,33 +35,46 @@ void CInternalTracer::traceInstanceData(const char *const paTypeName, const char
   fillStringsVector(paInternal, paInternalLength, internal);
   fillStringsVector(paInternalFBs, paInternalFBsLength, internalFBs);
 
-  mOutput.emplace_back("instanceData", std::make_unique<FBInstanceDataPayload>(paTypeName, paInstanceName, inputs, outputs, internal, internalFBs), getNanoSecondsMonotonic());
+  mEvents.emplace_back("instanceData", std::make_unique<FBInstanceDataPayload>(paTypeName, paInstanceName, inputs, outputs, internal, internalFBs), getNanoSecondsMonotonic());
 }
 
 void CInternalTracer::traceReceiveInputEvent(const char *const paTypeName, const char *const paInstanceName, const uint64_t paEventId)
 {
-  mOutput.emplace_back("receiveInputEvent", std::make_unique<FBInputEventPayload>(paTypeName, paInstanceName, paEventId), getNanoSecondsMonotonic());
+  mEvents.emplace_back("receiveInputEvent", std::make_unique<FBInputEventPayload>(paTypeName, paInstanceName, paEventId), getNanoSecondsMonotonic());
 }
 
 void CInternalTracer::traceSendOutputEvent(const char *const paTypeName, const char *const paInstanceName, const uint64_t paEventId)
 {
-  mOutput.emplace_back("sendOutputEvent", std::make_unique<FBOutputEventPayload>(paTypeName, paInstanceName, paEventId), getNanoSecondsMonotonic());
+  mEvents.emplace_back("sendOutputEvent", std::make_unique<FBOutputEventPayload>(paTypeName, paInstanceName, paEventId), getNanoSecondsMonotonic());
 }
 
 void CInternalTracer::traceInputData(const char *const paTypeName, const char *const paInstanceName,
                                      const uint64_t paDataId, const char *const paValue)
 {
 
-  mOutput.emplace_back("inputData", std::make_unique<FBDataPayload>(paTypeName, paInstanceName, paDataId, paValue), getNanoSecondsMonotonic());
+  mEvents.emplace_back("inputData", std::make_unique<FBDataPayload>(paTypeName, paInstanceName, paDataId, paValue), getNanoSecondsMonotonic());
 }
 
 void CInternalTracer::traceOutputData(const char *const paTypeName, const char *const paInstanceName, const uint64_t paDataId,
                                       const char *const paValue)
 {
-  mOutput.emplace_back("outputData", std::make_unique<FBDataPayload>(paTypeName, paInstanceName, paDataId, paValue), getNanoSecondsMonotonic());
+  mEvents.emplace_back("outputData", std::make_unique<FBDataPayload>(paTypeName, paInstanceName, paDataId, paValue), getNanoSecondsMonotonic());
 }
 
 bool CInternalTracer::isEnabled()
 {
   return true;
 }
+
+void CInternalTracer::fillStringsVector(const char *const *const paIn, const uint32_t paLen, std::vector<std::string> &paOut)
+{
```

Comment:
this bracket should be at the end of the line above. Please fix formating in this file

Reply:
done. Any objections to add some clang formatting check or some other tool?

Reply:
I use the formatting settings that we provide for Eclipse CDT. But I'm happy with anything making this more correkt.

Reply:
I added an issue for better tracking of the decisions https://github.com/eclipse-4diac/4diac-forte/issues/379

Reply:
thx